aws/resource_aws_route_table_test.go

```diff
@@ -252,9 +251,9 @@ func TestAccAWSRouteTable_tags(t *testing.T) {
 			{
 				Config: testAccRouteTableConfigTags,
 				Check: resource.ComposeTestCheckFunc(
-					testAccCheckRouteTableExists(resourceName, &route_table),
+					testAccCheckRouteTableExists(resourceName, &routeTable),
 					resource.TestCheckResourceAttr(resourceName, "tags.%", "1"),
-					resource.TestCheckResourceAttr(resourceName, "tags.foo", "bar"),
+					resource.TestCheckResourceAttr(resourceName, "tags.test", "bar"),
```

Comment:
It appears the test configuration wasn't changed here:

```
--- FAIL: TestAccAWSRouteTable_tags (33.75s)
testing.go:669: Step 0 error: Check failed: Check 3/3 error: aws_route_table.test: Attribute 'tags.test' not found
```

```suggestion
					resource.TestCheckResourceAttr(resourceName, "tags.foo", "bar"),
```

Reply:
fixed